docs/content/cache.md

```diff
@@ -182,6 +182,28 @@ and password) in your remote and it will be automatically enabled.
 Affected settings:
 - `cache-workers`: _Configured value_ during confirmed playback or _1_ all the other times
 
+##### Certificate Validation #####
+
+When the Plex server is configured to only accept secure connections, it is
+possible to use `.plex.direct` URL's to ensure certificate validation succeeds.
+These URL's are used by Plex internally to connect to the Plex server securely.
+
+The format for this URL's is the following:
+
+https://ip-with-dots-replaced.server-hash.plex.direct:32400/
+
+The `ip-with-dots-replaced` part can be any IPv4 address, where the dots
+have been replaced with dashes, e.g. `127.0.0.1` becomes `127-0-0-1`.
+
+To get the `server-hash` part, the easiest way is to visit
+
+https://plex.tv/api/resources?includeHttps=1&X-Plex-Token=your-plex-token
```

Comment:
Can you explain where to grab the Plex Token from as well?

Reply:
While you and I understand, as you mentioned, it won't hurt to make the documentation more complete by referring where to get each value, etc.

Reply:
Sure, but where to stop adding things like this to the documentation.

Anybody who wants to use Plex more than "install on my NAS, add files and play" should know how to find and use the Plex-Token, since it is the key to all further usage.

Finding these `.plex.direct` URL's is a bit harder and they are barely documented otherwise, so adding this information adds a real value for others.